src/internal/pjsdb/job_crud.go

```diff
@@ -162,7 +172,7 @@ func GetJob(ctx context.Context, tx *pachsql.Tx, id JobID) (Job, error) {
 	ctx = pctx.Child(ctx, "getJob")
 	record := jobRecord{}
 	err := sqlx.GetContext(ctx, tx, &record, selectJobRecordPrefix+`
-	WHERE j.id = $1 GROUP BY j.id;`, id)
+	WHERE j.id = $1 GROUP BY j.id, jc.job_hash;`, id)
```

Comment:
Why group by like this, can same j.id have different jc.job_hash?

Reply:
I'm not 100% sure what causes the group by clause to be enforced, but I did this to satisfy postgres otherwise we'd get an error.

Reply:
Oh yea, that's right. Columns that are not in the where clause have to be in the group by